gogoproto/gogo.proto

```diff
@@ -67,6 +67,8 @@ extend google.protobuf.FileOptions {
 	optional bool goproto_extensions_map_all = 63025;
 	optional bool goproto_unrecognized_all = 63026;
 	optional bool gogoproto_import = 63027;
+
+	optional bool validator_all = 63028;
```

Comment:
I originally thought about doing a validator for every file. But that's not needed since the `Validator` interface was declared.